s32/mcux/devices/S32Z270/S32Z270_glue_mcux.h

```diff
@@ -247,4 +247,45 @@
 /** Interrupt vectors for the LPI2C peripheral type */
 #define LPI2C_IRQS                               { RTU_LPI2C1_IRQn, RTU_LPI2C2_IRQn }
 
+/* EDMA - Peripheral instance base addresses */
+/** Peripheral EDMA3_0 base address */
+#define EDMA3_0_BASE                                IP_EDMA_0_MP_BASE
+/** Peripheral EDMA3_0 base pointer */
+#define EDMA3_0                                     ((DMA_Type *)EDMA3_0_BASE)
+/** Interrupt vectors for the DMA3_0 peripheral type */
+#define DMA3_0_IRQS                                 { RTU_DMA0_0_IRQn, RTU_DMA0_1_IRQn, RTU_DMA0_2_IRQn, RTU_DMA0_3_IRQn, RTU_DMA0_4_IRQn, RTU_DMA0_5_IRQn, RTU_DMA0_6_IRQn, RTU_DMA0_7_IRQn, RTU_DMA0_8_IRQn, RTU_DMA0_9_IRQn, RTU_DMA0_10_IRQn, RTU_DMA0_11_IRQn, RTU_DMA0_12_IRQn, RTU_DMA0_13_IRQn, RTU_DMA0_14_IRQn, RTU_DMA0_15_IRQn, RTU_DMA0_16_IRQn, RTU_DMA0_17_IRQn, RTU_DMA0_18_IRQn, RTU_DMA0_19_IRQn, RTU_DMA0_20_IRQn, RTU_DMA0_21_IRQn, RTU_DMA0_22_IRQn, RTU_DMA0_23_IRQn, RTU_DMA0_24_IRQn, RTU_DMA0_25_IRQn, RTU_DMA0_26_IRQn, RTU_DMA0_27_IRQn, RTU_DMA0_28_IRQn, RTU_DMA0_29_IRQn, RTU_DMA0_30_IRQn, RTU_DMA0_31_IRQn }
+#define DMA3_0_ERROR_IRQS                           { RTU_DMA0_ERR_IRQn }
+
+/** Peripheral EDMA3_1 base address */
+#define EDMA3_1_BASE                                IP_EDMA_1_MP_BASE
+/** Peripheral EDMA3_1 base pointer */
+#define EDMA3_1                                     ((DMA_Type *)EDMA3_1_BASE)
+/** Interrupt vectors for the DMA3_1 peripheral type */
+#define DMA3_1_IRQS                                 { RTU_DMA1_0_IRQn, RTU_DMA1_1_IRQn, RTU_DMA1_2_IRQn, RTU_DMA1_3_IRQn, RTU_DMA1_4_IRQn, RTU_DMA1_5_IRQn, RTU_DMA1_6_IRQn, RTU_DMA1_7_IRQn, RTU_DMA1_8_IRQn, RTU_DMA1_9_IRQn, RTU_DMA1_10_IRQn, RTU_DMA1_11_IRQn, RTU_DMA1_12_IRQn, RTU_DMA1_13_IRQn, RTU_DMA1_14_IRQn, RTU_DMA1_15_IRQn }
+#define DMA3_1_ERROR_IRQS                           { RTU_DMA1_ERR_IRQn }
+
+/** Peripheral EDMA3_4 base address */
+#define EDMA3_4_BASE                                IP_EDMA_4_MP_BASE
+/** Peripheral EDMA3_4 base pointer */
+#define EDMA3_4                                     ((DMA_Type *)EDMA3_4_BASE)
+/** Interrupt vectors for the DMA3_4 peripheral type */
+#define DMA3_4_IRQS                                 { RTU_DMA4_0_IRQn, RTU_DMA4_1_IRQn, RTU_DMA4_2_IRQn, RTU_DMA4_3_IRQn, RTU_DMA4_4_IRQn, RTU_DMA4_5_IRQn, RTU_DMA4_6_IRQn, RTU_DMA4_7_IRQn, RTU_DMA4_8_IRQn, RTU_DMA4_9_IRQn, RTU_DMA4_10_IRQn, RTU_DMA4_11_IRQn, RTU_DMA4_12_IRQn, RTU_DMA4_13_IRQn, RTU_DMA4_14_IRQn, RTU_DMA4_15_IRQn }
+#define DMA3_4_ERROR_IRQS                           { RTU_DMA4_ERR_IRQn }
+
+/** Peripheral EDMA3_5 base address */
+#define EDMA3_5_BASE                                IP_EDMA_5_MP_BASE
+/** Peripheral EDMA3_5 base pointer */
+#define EDMA3_5                                     ((DMA_Type *)EDMA3_5_BASE)
```

Comment:
why you are defining macros named EDMA3_, DMA3_, DMA_? what's with some many prefixes? please defined only what is needed following the template of mcux-sdk device definitions

Reply:
I redefined the macro name to `DMA_` and removed the unnecessary definitions.